### PR TITLE
November update post

### PR DESCRIPTION
**Pull request by batpigandme**
Post for November updates in the Tidyverse.

Hope you don't mind my adding you to the reviewer panel, @topepo— I just want to make sure I'm not misrepresenting anything in tidy-model-ville.

### CODE REVIEW

File: content/articles/2017-12-nov-updates-2017.Rmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+* rlang 0.1.4 ([release notes](https://github.com/tidyverse/rlang/releases))
+
+* tidyselect 0.2.3 ([release notes](https://github.com/tidyverse/tidyselect/releases))
+
+### tidyselect
+
+tidyselect provides a common back-end for `dplyr::select()`, `tidyr::gather()`, as well as for modelling packages. It is also the source of selection helpers, such as `everything()`, and `starts_with()`. tidyselect allows you to create selecting verbs that are consistent across tidyverse packages.
+
+### tidy models
```
Comment (topepo) [inline content/articles/2017-12-nov-updates-2017.Rmd]: I don't know that we've standardized on this name for those packages but that is good for now.

File: content/articles/2017-12-nov-updates-2017.Rmd [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+[yardstick](https://topepo.github.io/yardstick/) allows you to easily create tidy performance estimates. Using a syntax similar to dplyr’s you can compute common performance metrics, such as precision, and recall (for classification), or numeric metric outcomes for regression, and have them returned in a tidy data frame. 
+
+[recipes](https://topepo.github.io/recipes/) is an extensible framework for feature selection and the creation of preprocess design matrices, which can then be applied to statistical and machine learning models. The updated version of recipes includes a tidy method for many of the step functions. The tidy method returns relevant information about the step. This could include estimated parameters or which variables were affected by the step.
```
Comment (topepo) [inline content/articles/2017-12-nov-updates-2017.Rmd]: There were a bunch of new steps added too. Can you link to the NEW files for [`recipes`](https://topepo.github.io/recipes/news/index.html) and [`rsample`](https://topepo.github.io/rsample/news/index.html)?

File: content/articles/2017-12-nov-updates-2017.Rmd [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+[rsample](https://topepo.github.io/rsample/)’s major upgrade from caret is that it allows for *nested* resampling. The goal is to have a modular, extensible set of methods that can be used across R packages for traditional resampling techniques, and estimating model performance. rsample can be used to create objects containing resamples of the original data, allowing you to create a model and optimization parameters with placeholders for features to be defined later.
```
Comment (topepo) [inline content/articles/2017-12-nov-updates-2017.Rmd]: Maybe add

> The [package website](https://topepo.github.io/rsample/) has examples for resampling time series, survival models, and neural networks using `tensorflow`.

**Comment by topepo** [comment]
That's all of my comments. Thanks

File: content/articles/2017-12-nov-updates-2017.Rmd (hunk 3 above)
Comment (batpigandme) [reply content/articles/2017-12-nov-updates-2017.Rmd]: Yeah, this was written before mine eyes did fall upon a bunch of your best new material!

**Comment by hadley** [comment]
Oops, I forgot about this one. Can you please make it November and December updates? I don't think anything happened in Dec, so shouldn't require any changes?

For future reference, once the post has been reviewed, you should feel free to merge yourself (after checking that the dates have been updated)

**Comment by batpigandme** [comment]
@hadley changes made. I got rid of "happenings" just because "November/December" is already so long character-wise... LMK if this displeases you. 😤

**Comment by hadley** [comment]
Perfect - please publish and tweet!

**Pull request merged**